.github/workflows/cicd.yml

```diff
@@ -69,6 +69,7 @@ jobs:
     uses: ./.github/workflows/reusable-test.yml
     with:
       name: unit_tests
+      pre_test: appverif -enable Exceptions Handles Heaps Leak Locks Memory SRWLock Threadpool TLS DangerousAPIs DirtyStacks TimeRollOver -for unit_tests.exe
```

Comment:
I think you should do this for all the user mode test apps including but not limited to api_tests.exe ?

Reply:
I believe @saxena-anurag mentioned only unit_tests. I did not see that api_tests.exe was included in cicd.yml pipeline. However, I think I need to include the app verifier in unit_tests_leak_detection, netebpf_ext_unit_tests, and bpf2c, sanitize_unit_tests, and stress, and  low_memory.

Reply:
api_test.exe is included in the cicd pipeline.  The "driver" job in cicd.yml has
```
test_command: .\execute_ebpf_cicd_tests.ps1
```
execute_ebpf_cicd_tests.ps1 calls Invoke-XDPTestsOnVM which is in vm_run_tests.psm1.
That calls Invoke-CICDTests which is in run_driver_tests.psm1 and that invokes:
"api_test.exe", "bpftool_tests.exe", "sample_ext_app.exe", and "socket_tests.exe".

Reply:
I opened [a separate issue](https://github.com/microsoft/ebpf-for-windows/issues/1973) only for those tests and took separate action items to address them since they are not directly called in cicd.yml file.